<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Humiture</class>
 <widget class="QWidget" name="Humiture">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Humiture</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/background.jpg);</string>
  </property>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>651</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>16</horstretch>
     <verstretch>9</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	border-image: url(:/image/title.png);
}

QLabel#label_logo {
	border-image: url(:/image/logo.png);
}
QPushButton#pushButton_min{
	border-image: url(:/image/min.png);
}
QPushButton#pushButton_max{
	border-image: url(:/image/max.png);
}
QPushButton#pushButton_close{
	border-image: url(:/image/close.png);
}</string>
   </property>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>39</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>39</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/max.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>39</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/display.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/com.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/modbus.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/read.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/write.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
